card/mod/bootstrap/set/type_plus_right/customized_skin/colors.rb

```diff
@@ -13,30 +13,32 @@
 # TODO: deal with that
 
 def variable_value name
```

Comment:
maybe a comment that "name" can SCSS (and is treated as such if it starts with a "$")?